scripts/tags/exturl.js

```diff
@@ -0,0 +1,57 @@
+/* global hexo */
+// Usage: {% exturl text url "title" %}
+// Alias: {% extlink text url "title" %}
+
+'use strict';
+
+var util = require('hexo-util');
```

Comment:
@iissnan «external link» tag work like standart «link» tag in hexo tag plugins: 
https://hexo.io/docs/tag-plugins.html#Link
https://github.com/hexojs/hexo/blob/master/lib/plugins/tag/link.js

So, if standart tags need dependiens for hexo-util, this «external link» tag need it too.
But, i don't install anything from last year and standart «link» tag worked fine as «external link» tag too.